tests/classification/test_accuracy.py

```diff
@@ -103,6 +103,11 @@ def test_accuracy_fn(self, preds, target, subset_accuracy):
             },
         )
 
+    def test_differentiability(self, preds, target):
+        metric = Accuracy()
```

Comment:
to simulate this correctly we need tensors with grad enabled 
```suggestion
        preds.requires_grad = True
        metric = Accuracy()
```